src/main/java/gov/nasa/jpl/mbee/mdk/systems_reasoner/actions/SpecializeStructureAction.java

```diff
@@ -113,8 +113,12 @@ public Classifier createSpecialClassifier(Namespace container, List<RedefinableE
         }
         Utils.createGeneralization(classifier, specific);
 
-
-        for (final NamedElement ne : specific.getInheritedMember()) { // Exclude Classifiers for now -> Should Aspect Blocks be Redefined?
+        Set<NamedElement> listOfAllMembers = new HashSet<>();
```

Comment:
`Set<NamedElement> listOfAllMembers = new HashSet<>(specific.getInheritedMember());` can replace this block.